go.mod

```diff
@@ -7,8 +7,9 @@ module github.com/openshift-kni/cnf-features-deploy
 //   - openshift-ci/Dockerfile*
 //   - ztp/resource-generator/Containerfile
 //   - ztp/tools/pgt2acmpg/go.mod
-go 1.22
-toolchain go1.22.5
+go 1.22.4
```

Comment:
this is weird no?

the tool version newer than the go version?

Reply:
I don't know. It is changed by `go mod tidy`.

But there is no reason to commit that bump in this PR, so I reverted those two lines.